lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -23,281 +23,289 @@
                     'supported_by': 'community'}
 
 
-DOCUMENTATION = '''
+DOCUMENTATION = r'''
```

Comment:
That's a raw string, it's not strictly necessary here, but we want tostandardize this.
This is still open for discussion BTW  but in this context it shouldn't really matter.